aws/rust-runtime/aws-config/src/imds/credentials.rs

```diff
@@ -234,17 +235,19 @@ impl ImdsCredentialsProvider {
                 access_key_id,
                 secret_access_key,
                 session_token,
+                account_id,
```

Comment:
This isn't expected from this IMDS endpoint though is it? Should we keep IMDS separate and do in a later PR as the SEP is still in-progress for this?

Reply:
Makes sense. Reverted (and left `TODO`) handling account ID for this place in a032dca.